zap/src/main/java/org/parosproxy/paros/extension/manualrequest/http/impl/HttpPanelSender.java

```diff
@@ -3,7 +3,7 @@
  *
  * ZAP is an HTTP/HTTPS proxy for assessing web application security.
  *
- * Copyright 2012 The ZAP Development Team
+ * Copyright 2020 The ZAP Development Team
```

Comment:
It's not necessary to change it.

Reply:
spotlessApply does that sometimes, I haven’t figured out the trigger...

Reply:
Ah, it was because of the "ZAP" comment (not needed in this class).

Reply:
Yea it was done by spotlessApply, should I change it back then?

Reply:
Yes please